website/content/docs/secrets/databases/index.mdx

```diff
@@ -104,75 +105,157 @@ TTL.
+1. Click on Database plugin dropdown and select desired plugin ie. **postgreSQL**
+
+![Partial screenshot of the Vault GUI showing the populated "Plugin config" section](/img/gui/databases/pluginConfig.png)
+
+1. Fill in the chosen plugin's specific fields
+1. Click create database to create the database connection
+
+</Tab>
+
+</Tabs>
+
+After configuring the root user, it is highly recommended you rotate that user's
+password such that the vault user is not accessible by any users other than
+Vault itself:
+
+```shell-session
+$ vault write -force database/rotate-root/my-database
+```
+
+!> When this is done, the password for the user specified in the previous step
+is no longer accessible. Because of this, it is highly recommended that a
+user is created specifically for Vault to use to manage database
+users.
 
-   The `{{username}}` and `{{password}}` fields will be populated by the plugin
-   with dynamically generated values. In some plugins the `{{expiration}}` field is also supported.
+<Tabs>
+
+<Tab heading="CLI" group="cli">
+
+Configure a role that maps a name in Vault to a set of creation statements to
+create the database credential:
+
+```shell-session
+$ vault write database/roles/my-role \
+    db_name=my-database \
+    creation_statements="..." \
+    default_ttl="1h" \
+    max_ttl="24h"
+Success! Data written to: database/roles/my-role
+```
+
+The `{{username}}` and `{{password}}` fields will be populated by the plugin
+with dynamically generated values. In some plugins the `{{expiration}}` field is also supported.
+
+</Tab>
+
+<Tab heading="GUI"  group="gui">
+
+1. From Secret Engines List, click on **database/**
+1. Click on **Roles** tab and click **Create role +**
+
+![Screenshot of the Vault GUI showing the "Create Role" page](/img/gui/databases/roleDefaultState.png)
+
+1. Enter desired name of role in **Role name** field
+1. In the **Connection name** dropdown, choose the desired DB connection for this role to be created in
+1. In the **Type of role** dropdown, choose between creating a static or dynamic role
+
+#### Creating a Dynamic Role
+
+![Partial screenshot of the Vault GUI showing the "Role settings" section for a Dynamic role](/img/gui/databases/roleDynamic.png)
+
+1. Manage desired role's TTL and Max TTL expiration or keep defaults
+1. Enter items as desired in statements fields or leave blank
+1. Click **Create role** to create the role
+
+#### Creating a Static Role
+
+![Partial screenshot of the Vault GUI showing the "Role settings" section for a static role](/img/gui/databases/roleStatic.png)
+
+1. Enter a username from the database that this role will be mapped to
+1. Enter a password for this role if desired or leave blank
+1. Specify the **Rotation period of this role**, or keep the default of 24 hours
+1. Toggle whether or not to rotate this roles password immediately or to bypass the first rotation
+1. Click **Create role** to create the role
```

Comment:
```suggestion
```

I think we can simplify these instructions instead of adding two additional sub-sections